Sources/Website/HomePage.swift

```diff
@@ -112,7 +114,6 @@ extension HomePage {
             Div {
                 H5("Powering companies like:").class("used-by-caption")
                 Div {
-                    CompanyCard(name: "Emerge Tools", url: "https://emergetools.com", logo: "/images/emerge-tools.png")
```

Comment:
How come this was removed? Do we have anything to replace it?

Reply:
It didn't appear on the page and isn't a company that actually uses it, they just sponsor